### PR TITLE
External CDDL files and include for #244

### PR DESCRIPTION
**Pull request by aj-stein-nist**
This change externalizes the two normative definitions for transparent and signed statements, respectively, and adds them to the final draft with the kramdown `includes` directive. This change allows developers to use a git sub-module or programmatically use the git version of the spec to have utilities for code generation and other capabilities use the definitions without more cumbersome extraction routines for the source Markdown or RFC XML representations of the spec.

Below is a local eval copy of the HTML representation of the draft with the `include`d files properly integrated.

image

### CODE REVIEW

File: signed_statement.cddl [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (SteveLasker) [inline signed_statement.cddl:1]:
```suggestion
```

File: transparent_statement.cddl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Transparent_Statement = #6.18(COSE_Sign1)
+
+Unprotected_Header = {
+  &(receipts: 394)  => [+ Receipt]
+}
```
Comment (SteveLasker) [inline transparent_statement.cddl:5]:
```suggestion
}

```

**Review by SteveLasker — APPROVED** [review]
I was expecting this to require changes to `make`, but this just worked. Thanks for the update, @aj-stein-nist 
LGTM with two line nits

**Comment by aj-stein-nist** [comment]
It seems you cannot batch suggestions on mobile GitHub app. I'll do it later this evening.

**Pull request merged**